### DISCUSSION TITLE
500: Internal Server Error When Register New User

### DISCUSSION THREAD

**Discussion by Antonimo-Dev** [wallabag/wallabag#7658] [Q&A]
I have just installed on cPanel using Scriptaculous on shared hosting. I am able to log in using Brave browser and the browser extension is successfully saving articles.

When I try to register a new user from the front end (https://wallabag.DOMAIN.com/web/register) I am receiving a 500: Internal Server Error

![capture_2024-09-03_161412](https://github.com/user-attachments/assets/b7b62aaa-63e7-486c-8dfa-f81e70d6db93)

Can anybody suggest a solution please?

**Comment by j0k3r** [comment]
Looks like an issue while sending the activation email.
Can you check `app/config/parameters.yml`? They should be sth about 127.0.0.1:465 and that value isn't working on your server.

**Comment by Antonimo-Dev** [comment]
Hi,

The closest I can find in  app/config/parameters.yml is:

`mailer_dsn: 'smtp://127.0.0.1'`

The parameters.yml is attached.
[parameters.zip](https://github.com/user-attachments/files/16853289/parameters.zip)

**Reply by j0k3r** [reply]
I've no idea then.
You might need to check on cPanel or Scriptaculous PHP config, that error is really unrelated to wallabag but to PHP I think.

**Reply by Antonimo-Dev** [reply]
Thanks for your reply. I am checking with my host. I'll post back any useful information.
